scripts/check_branch.py

```diff
@@ -21,9 +21,6 @@ def check_branch(branch: Optional[str], stage: str) -> None:
     RuntimeError: Non-protected branch 'feature/foo' can't be deployed to main deployment 'prod'
 
     >>> check_branch('staging', 'hannes.local')
```

Comment:
```suggestion
    >>> check_branch('develop', 'hannes.local')
```

Since that's our main use case. Or maybe have one for develop and one for staging.